lib/KBaseReport/KBaseReportImpl.pm

```diff
@@ -1,1048 +0,0 @@
-package KBaseReport::KBaseReportImpl;
```

Comment:
PR looks good, there is only one thing I feel uncomfortable. It's support for direct calls to internal KBaseReportImpl code here: https://github.com/scanon/RAST_SDK/blob/29149a297231d40a86859e14842b622611590f99/lib/Bio/KBase/kbaseenv.pm#L48-L52
Since you deleted it we may consider deleting support for it (and in config parameter switching mode for it too: https://github.com/scanon/RAST_SDK/blob/29149a297231d40a86859e14842b622611590f99/deploy.cfg#L61).